docs/troubleshooting-oauth2.md

```diff
@@ -23,119 +23,116 @@ limitations under the License.
 
 ```
 OAuth resource [rabbitmq] not available. OpenId Discovery endpoint https://<the_issuer_url>/.well-known/openid-configuration not compliant
 ```
 
-#### Troubleshoot the issue
-
-This error means that the endpoint does not return a JSON payload which matches with the [OpenId Connect Discovery Configuration](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig) format.
-
-There are several the possible causes:
-
- * The payload returned by the endpoint is not compliant because it is empty or it is missing some critical information. To identify the root cause, you open the browser's javascript console and search for one of these possible error messages:
-  - `Payload does not contain openid configuration` This error occurs when the payload is empty or it is not a JSON payload
-  - `Missing authorization_endpoint` This error occurs when the JSON attribute `authorization_endpoint` is missing
-  - `Missing token_endpoint` This error occurs when the JSON attribute `token_endpoint` is missing
-  - `Missing jwks_uri` This error occurs when the JSON attribute `jwks_uri` is missing
-* The url is incorrect. Contact the maintainers of the Identity Provider to get the correct URL for the OpenID Connect discovery endpoint
+#### Troubleshooting
 
+This issue is caused when the endpoint is not returning a JSON payload which matches with the [OpenId Connect Discovery Configuration](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig).
+These are the possible causes:
+- The payload returned by the endpoint is not compliant because it is empty or it is missing some critical information. To identify the root cause, open the browser's javascript console and search for one of these possible error messages:
+  - `Payload does not contain openid configuration` This error occurs when the payload is empty or it is not a JSON payload.
+  - `Missing authorization_endpoint` This error occurs when the JSON attribute `authorization_endpoint` is missing.
+  - `Missing token_endpoint` This error occurs when the JSON attribute `token_endpoint` is missing.
+  - `Missing jwks_uri` This error occurs when the JSON attribute `jwks_uri` is missing.
+- The url is wrong. Check out with your administrator of your Identity Provider to get the correct url to the OpenId Connect Discovery endpoint.
 
 ### Not authorized {#not-authorized-error}
 
-#### Steps to reproduce the issue
+#### Steps to reproduce
 
-Open the root url of the management UI in the browser, click on the buttoh "Click here to logon" and enter the credentials requested by your Identity Provider. You are redirected back to the management UI with the following error:
+Open the root url of the management UI in the browser, click on the button "Click here to logon" and enter the credentials requested by your Identity Provider. You are redirected back to the management UI with the following error:
 
```

Comment:
add full stop after the word "browser"
Then start a new sentence written as: "click the "Click here to logon" button and enter the credentials requested by your Identity Provider."